tests/data/spe1_st/tests/conftest.py

```diff
@@ -2,17 +2,17 @@
 import pytest
 
 
-@pytest.fixture(scope="session")
-def get_info():
+@pytest.fixture(scope="session", autouse=True)
+def start_storage_server():
```

Comment:
I would refactor it a bit:

```python
@pytest.fixture(scope="session", autouse=True)
def start_storage_server():
    from ert.shared.services import Storage
    with Storage.start_server() as service:
        service.wait_until_ready(timeout=20)
        yield service
    service.shutdown()
```

Reply:
Shouldn't the service shutdown when the context closes ? Is it necessary to have one explicitly doing it?

Reply:
That's right it is not needed